src/main/java/com/jalasoft/jfc/model/strategy/CommandImageMagickPath.java

```diff
@@ -28,7 +28,7 @@
     private  String imageMagickPath;
 
     /**
-     * Gets the Image Magick Path.
+     * Assigns the Image Magick Path.
```

Comment:
you could improve "Assigns the Image Magick Path" to **Assigns Image Magick's Path.**